style/americana.js

```diff
@@ -155,11 +182,15 @@ var map = new maplibregl.Map({
   zoom: 4, // starting zoom
   attributionControl: false,
 });
+
+map.on("styleimagemissing", function (e) {
+  Shield.missingIconLoader(map, e);
+});
+
 map.addControl(
   new maplibregl.AttributionControl({
     customAttribution:
       '<a href="https://openmaptiles.org/" target="_blank">&copy; OpenMapTiles</a> <a href="https://www.maptiler.com/copyright/" target="_blank">&copy; MapTiler</a> <a href="https://www.openstreetmap.org/copyright" target="_blank">&copy; OpenStreetMap contributors</a>',
   })
 );
 map.addControl(new maplibregl.NavigationControl(), "top-left");
-document.querySelector("#map canvas").focus();
```

Comment:
Deliberate?

Also: `map.getCanvas().focus()`.

Reply:
Looks like a merge error, I've reverted that delete.